src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,35 @@
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import usersFromServer from '../../api/users.json';
+import commentsFromServer from '../../api/comments.json';
```

Comment:
Ensure that the `comments.json` file exists in the `api` directory and contains the necessary comment data for filtering post comments.